tests/invert_test_gtest.hpp

```diff
@@ -126,15 +136,21 @@ std::string gettestname(::testing::TestParamInfo<test_t> param)
   name += get_solve_str(::testing::get<2>(param.param)) + std::string("_");
   name += get_prec_str(::testing::get<3>(param.param));
   if (::testing::get<4>(param.param) > 1)
-    name += std::string("_shift_") + std::to_string(::testing::get<4>(param.param));
-  if (::testing::get<5>(param.param) > 1)
-    name += std::string("_solution_accumulator_pipeline_") + std::to_string(::testing::get<5>(param.param));
+    name += std::string("_shift") + std::to_string(::testing::get<4>(param.param));
+  if (::testing::get<5>(param.param) > 1)// && support_solution_accumulator_pipeline(::testing::get<0>(param.param)))
```

Comment:
what about the commented part here?

Reply:
Erroneous inclusion.  Fixed with [cdc47f9](https://github.com/lattice/quda/pull/1318/commits/cdc47f9163faa35e24b29d43bfe32de6da981c4c)